pkg/util/vault.go

```diff
@@ -55,11 +60,36 @@ func NewVaultClient(pi *PodInfo) (*vaultapi.Client, error) {
 	}
 
 	binding := app.DeepCopy()
```

Comment:
`binding := app.DeepCopy()` not needed anymore. Since we don't modify the original app binding anymore.